src/main/java/spoon/reflect/visitor/DefaultJavaPrettyPrinter.java

```diff
@@ -1773,4 +1795,35 @@ public void calculate(CompilationUnit sourceCompilationUnit, List<CtType<?>> typ
 	public Map<Integer, Integer> getLineNumberMapping() {
 		return printer.getLineNumberMapping();
 	}
+
+	/**
+	 * @return current implementation of {@link PrinterTokenWriter}
+	 */
+	public PrinterTokenWriter getPrinterTokenWriter() {
```

Comment:
do we need a getter? for encapsulation of configuration, the setter only is fine

Reply:
ok, I will remove it